tests/config_test.rs

```diff
@@ -0,0 +1,18 @@
+use tonlib::cell::BagOfCells;
```

Comment:
I would recommend to move this test to client_test.rs to preserve consistency

Reply:
I will fix this